config.go

```diff
@@ -337,7 +337,7 @@ func loadConfig() (*config, []string, error) {
 		BlockMaxSize:      defaultBlockMaxSize,
 		BlockPrioritySize: defaultBlockPrioritySize,
 		SigCacheMaxSize:   defaultSigCacheMaxSize,
-		MaxOrphanTxs:      maxOrphanTransactions,
+		MaxOrphanTxs:      1000,
```

Comment:
This should be a `defaultMaxOrphanTransactions` constant to be consistent with the surrounding code.